src/collections/programs/layer5/index.mdx

```diff
@@ -0,0 +1,7 @@
+---
+title: 'Layer5'
+thumbnail: ./layer5.svg
+permalink: "https://layer5.io/careers/internships"
+redirect: "https://layer5.io/careers/internships"
```

Comment:
remove this redirect and update the PR, then I will check what else needs to be done.

Reply:
Okay

Reply:
All done